core/flyout_base.js

```diff
@@ -32,8 +32,10 @@ goog.require('Blockly.Events.BlockCreate');
 goog.require('Blockly.Events.VarCreate');
 goog.require('Blockly.FlyoutButton');
 goog.require('Blockly.Gesture');
+goog.require('Blockly.Scrollbar');
 goog.require('Blockly.Tooltip');
 goog.require('Blockly.Touch');
+goog.require('Blockly.blockRendering');
```

Comment:
Move this up. Should be alphabetical ignoring case.

Reply:
Updated.